Components/ChartItem/index.js

```diff
@@ -8,10 +8,18 @@ const ImageContainer = styled.div`
   justify-content: center;
 `;
 
-export default function ChartItem({ name, icon, onSelectChartType }) {
+export default function ChartItem({
+  name,
+  icon,
+  onSelectChartType,
+  clickedChartType,
+}) {
+  function compareChartTypeName() {
+    return clickedChartType === name;
+  }
   return (
     <Container $direction="column">
-      <Button $variant="selected" onClick={onSelectChartType}>
+      <Button isActive={compareChartTypeName()} onClick={onSelectChartType}>
```

Comment:
does this need to be a function? 

This would get rid of 3 lines of code:

```suggestion
      <Button isActive={clickedChartType === name} onClick={onSelectChartType}>
```

Reply:
First I have made it like your suggestion but then we have discussed in the group whether we are supposed to write a function :)